src/cordova/android/OutlineAndroidLib/outline/src/main/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -251,7 +227,7 @@ private ErrorCode startTunnel(final TunnelConfig config) {
   private synchronized ErrorCode startTunnel(
       final TunnelConfig config, boolean isAutoStart) {
```

Comment:
I'd like to reduce the dependency on the TunnelConfig and be more explicit about what's needed here:
```suggestion
  private synchronized ErrorCode startTunnel(
      String id, String proxyConfigString, boolean isAutoStart) {
```

In fact, can we get rid of TunnelConfig and makeTunnelConfig for now?